src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -764,17 +775,41 @@ private function createProductAttribute($type, $name, $code = null)
     }
 
     /**
+     * @param string $type
+     * @param string $name
+     * @param string|null $code
+     *
+     * @return ProductAttributeInterface
+     */
+    private function provideProductAttribute($type, $name, $code = null)
+    {
+        if (null === $code) {
+            $code = StringInflector::nameToCode($name);
+        }
+
+        /** @var ProductAttributeInterface $productAttribute */
+        $productAttribute = $this->productAttributeRepository->findOneBy(['code' => $code]);
+        if (null !== $productAttribute) {
+            return $productAttribute;
+        }
+
+        return $this->createProductAttribute($type, $name, $code);
```

Comment:
I'm not sure about it. Shouldn't we create an attribute before first usage of it?